internal/serverinstall/k8s.go

```diff
@@ -633,6 +621,203 @@ func (i *K8sInstaller) InstallFlags(set *flag.Set) {
 	})
 }
 
+// Uninstall is a method of K8sInstaller and implements the Installer interface to
+// remove a waypoint-server statefulset and the associated PVC and service from
+// a Kubernetes cluster
+func (i *K8sInstaller) Uninstall(ctx context.Context, opts *InstallOpts) error {
+	ui := opts.UI
+	log := opts.Log
+
+	sg := ui.StepGroup()
+	defer sg.Wait()
+
+	s := sg.Add("Inspecting Kubernetes cluster...")
+	defer s.Abort()
```

Comment:
You gotta wrap this in a `func` or it'll abort just the first step created. You do this correctly in other implementations so probably just an easy miss. 😄